### PR TITLE
py3k: Use new-style division

### PR DESCRIPTION
**Pull request by QuLogic**
On Python 3, division returns floats, while `//` returns integers. In cases where there are explicit casts to `float`, or `float` literals, I have left the plain `/`. In cases where there are explicit casts to `int`, I've changed it to `//`.

I also assumed that bboxes, widths, and heights are all integers and used `//`, but if that's not the case, I can re-check those cases.

I'll add a few comments where I'm not sure.

### CODE REVIEW

File: gui/colors/adjbases.py [hunk 1]
```diff
@@ -876,10 +876,10 @@ def render_background_cb(self, cr, wd, ht):
         cr.set_line_width(self.OUTLINE_WIDTH)
         cr.stroke()
 
-        cr.rectangle(b, b, int(eff_wd / 2), eff_ht)
+        cr.rectangle(b, b, int(eff_wd // 2), eff_ht)
         cr.set_source_rgb(*curr.get_rgb())
         cr.fill()
-        cr.rectangle(wd / 2, b, eff_wd - int(eff_wd / 2), eff_ht)
+        cr.rectangle(wd // 2, b, eff_wd - int(eff_wd // 2), eff_ht)
```
Comment (QuLogic) [inline gui/colors/adjbases.py]: `cr.rectangle` can take a float, so I'm not sure if it should be an integer or not.

Comment (achadwick) [inline gui/colors/adjbases.py]: Our Cairo drawing often uses 0.5px offsets to make 1px lines align to the pixel grid. The worst that can result is a fuzzy line though.

Comment (QuLogic) [inline gui/colors/adjbases.py]: So should I put it back to float division, then? I'm not sure whether `wd` is or is not a float at this point.

**Comment by odysseywestra** [comment]
@QuLogic could you rebase this and resolve the conflicts on this one before we merge it in?

**Comment by QuLogic** [comment]
Rebased.

**Comment by odysseywestra** [comment]
@QuLogic  Thank you.
@achadwick Anything you want to address before merging this in? Like the comment you had in `gui/colors/adjbases.py `?

**Comment by achadwick** [comment]
Thanks for the rebase, and I'm sorry this has taken so long. Pushed.

I've been testing it out, and the colour selectors all look nice and sharp. No exceptions thrown so far :smile:

**Pull request merged**